quipucords/api/source/model.py

```diff
@@ -53,6 +53,8 @@ class Source(BaseModel):
     most_recent_connect_scan = models.ForeignKey(
         "api.ScanJob", null=True, on_delete=models.SET_NULL, related_name="+"
     )
+    http_proxy = models.CharField(max_length=255, null=True, blank=True)
+    https_proxy = models.CharField(max_length=255, null=True, blank=True)
```

Comment:
`:interestingly-intrigued:` I couldn't help but notice this. Do we need this distinction?

AFAIK, we don't support singular sources with multiple protocols in use. If the host is using https, then proxy HAS to be https. if its HTTP, then http.